extensions/rapidcap/src/types.ts

```diff
@@ -0,0 +1,11 @@
+interface Card {
+  question: string;
+  answer: string;
+  tag: string;
+}
+
+interface Preferences {
```

Comment:
Instead of defining your own interface, the raycast CLI should've generated a `raycast-env.d.ts` file which includes an autogenerated `Preferences` type

Reply:
Well yeah, I see that now 😆